static/styles/left_sidebar.css

```diff
@@ -190,25 +191,96 @@ li.show-more-topics {
     }
 }
 
-#private-container,
-#stream-filters-container {
+#left_sidebar_scroll_container {
+    outline: none;
     overflow-x: hidden;
     overflow-y: auto;
     position: relative;
     z-index: 0;
     width: 100%;
 }
 
-#stream-filters-container .simplebar-content-wrapper {
-    outline: none;
+#private_messages_container {
+    margin-bottom: $sections_vertical_gutter;
+    margin-right: 12px;
+
+    #toggle_private_messages_section_icon {
+        opacity: 0.7;
+        margin-left: 3px;
+        min-width: 13px;
```

Comment:
Does this 13px have anything to do with other variables, like `topic_resolve_width`?